react/src/lib/components/EdsIcon/components/EdsIcon.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import PropTypes from "prop-types";
+import * as edsIcons from "@equinor/eds-icons";
+import { IconData } from "@equinor/eds-icons";
+import { Icon } from "@equinor/eds-core-react";
+import { Tooltip } from "@material-ui/core";
+
+type IconProps = {
+    icon: string;
+    size?: 16 | 24 | 32 | 40 | 48;
+    color?: string;
+};
+
+export const EdsIcon: React.FC<IconProps> = (props) => {
```

Comment:
I'm a complete noob when it comes to typescript, and haven't worked much with React either, so this review is probably at most a chance for me to learn something, but anyways - I would have expected the `props` lambda function argument to be typed, but I suspect that the type is indirectly encoded in `React.FC<IconProps>`? : )

Reply:
Yes, that is correct.

Reply:
cool, thanks!